doc_user/UserManual.adoc

```diff
@@ -256,7 +256,7 @@ The OPD Extended module affords the following improvements:
 * Calendar to choose the visit date (1);
 * OPD number now is visible (2): it is the progressive number per year automatically set by the system, one can change the value needed. The OPD number MUST be unique for each year;
 * Registered Patient (3): before choosing a disease, you MUST first select a patient from the drop-down list; you can even fast search the patient with the search field nearby by typing part of its name or code; this will improve reliability for reports and statistics, and thus service to the patients by knowing his/her history.
-* Once the patient is selected and a previous OPD visit is found, special fields will be populated with the most recent visit of the patient (4), thus his/her personal data (5), the re-attendance checkbox will be selected automatically (6);
+* Once the patient is selected and a previous OPD visit is found, special fields will be populated with the most recent visit of the patient (4), thus his/her personal data (5), the readmittance checkbox will be selected automatically (6);
```

Comment:
Here the term "Re-Attendance" is the one commonly used to refer to OPD cases: it indicates a subsequent (visit) concerning the same medical case (same disease) or a planned followup.

Reply:
Readmittance is a more appropriate English word for "re-attendance" (which BTW is only correct without the "hyphen") and I have been changing it in the resource bundle files.

Reply:
done